Develop

zynkd opened the pull request:
[DEMO LINK](https://zynkd.github.io/react_movies-list-add-form/)

Anastasiia-Svintsova commented on src/components/NewMovie/NewMovie.tsx:
```diff
@@ -1,45 +1,106 @@
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+
+  const [title, setTitle] = useState('');
```
Redundant space
```suggestion
  const [count, setCount] = useState(0);
  const [title, setTitle] = useState('');
```

Anastasiia-Svintsova commented on src/components/NewMovie/NewMovie.tsx:
```diff
@@ -1,45 +1,106 @@
+
+    onAdd(newMovie);
+
+    // Form reset after submission
```
Remove comments

Anastasiia-Svintsova reviewed (changes requested):
- I can see errors on blur only for URL inputs. Add validation for all required inputs, please
<img width="742" alt="image" src="https://user-images.githubusercontent.com/99403198/233952656-ad4d455c-f0a9-4a14-b883-39abd09caf97.png">

Look at the video. I don't see any errors, but the button is still disabled 

https://user-images.githubusercontent.com/99403198/233953151-e81fdcd2-d5dd-465f-81ac-753998e184b5.mov

zynkd commented:
Hi Anastasiia, sorry, I was testing a few things and fogot to push it before I deployed it. Let me know if it looks good now - works on my end. Thank you!

qoa11a reviewed (changes requested):
You should remove the error message when the user changes the text of the input.
<img width="1582" alt="image" src="https://user-images.githubusercontent.com/102293862/234662047-b11d2032-80d6-4533-b3d3-03f8d09e74e4.png">

zynkd commented:
Hi, thank you for your previous comment. Not sure why I added validation func. on Title and ID. Removed both lines, all should work now. Let me know!

anstsot reviewed (changes requested):
Found an issue to fix. The validation works odd 

https://github.com/mate-academy/react_movies-list-add-form/assets/22824431/927a7772-02de-4d71-a96f-b6664c83e1a2

zynkd commented:
Hi @anstsot sorry about this - I forgot to deploy the new built to Github pages. Can you please check now? Thank you!

witflash reviewed (approved):
Good job! 👍
